### ISSUE TITLE
validation of non-human template file structure

### ISSUE THREAD

**Issue by kaitj** [afids/afids-validator#87]
Creating this issue so that it can be tracked.

## The problem

per @jclauneuro "Validate .fcsv structure template" was not working with no error messages. Seemed to work with human templates. I have added a non-human template for testing, and the non-human template appears to be validated properly. 

Would you be able to provide some additional information @jclauneuro?

## Environment

_as tested by @kaitj_ 
* Browser (with version) - Chrome v83.0.4103.61 (64-bit) 
* Dropdown menu option - "Validate .fcsv structure template"
* File uploaded (if applicable) - "macaqueMNI_mean.fcsv" (renamed as valid_nhp.fcsv)

_PR template was adopted from [appium](https://github.com/appium/appium/blob/master/.github/ISSUE_TEMPLATE.md)_

**Comment by jclauneuro** [comment]
Hey @tkai, I think you've already found one but for prosperity there are a bunch here including a macaqueMNI one https://github.com/afids/afids-macaca/blob/master/data/PHASE1_output_afid/macaqueMNI_MEAN.fcsv

**Comment by jclauneuro** [comment]
Here is a test case that was failing for me on the default but not with a human template selected. Couldn't attach a csv file so including inline. Also uploaded on Slack.

```
# Markups fiducial file version = 4.10
# CoordinateSystem = 0
# columns = id,x,y,z,ow,ox,oy,oz,vis,sel,lock,label,desc,associatedNodeID
vtkMRMLMarkupsFiducialNode_0,0.000,0.495,-0.537,0.000,0.000,0.000,1.000,1,1,0,1,AC,vtkMRMLScalarVolumeNode1
vtkMRMLMarkupsFiducialNode_1,-0.264,-12.505,0.673,0.000,0.000,0.000,1.000,1,1,0,2,PC,vtkMRMLScalarVolumeNode1
vtkMRMLMarkupsFiducialNode_6,-0.264,-18.755,-2.501,0.000,0.000,0.000,1.000,1,1,0,3,infracollicular sulcus,vtkMRMLScalarVolumeNode1
vtkMRMLMarkupsFiducialNode_11,-0.264,-10.761,-9.186,0.000,0.000,0.000,1.000,1,1,0,4,PMJ,vtkMRMLScalarVolumeNode1
vtkMRMLMarkupsFiducialNode_12,-0.308,-7.582,-5.186,0.000,0.000,0.000,1.000,1,1,0,5,superior interpeduncular fossa,vtkMRMLScalarVolumeNode1
vtkMRMLMarkupsFiducialNode_13,6.195,-13.145,-4.186,0.000,0.000,0.000,1.000,1,1,0,6,R superior LMS,vtkMRMLScalarVolumeNode1
vtkMRMLMarkupsFiducialNode_14,-6.509,-13.145,-4.513,0.000,0.000,0.000,1.000,1,1,0,7,L superior LMS,vtkMRMLScalarVolumeNode1
vtkMRMLMarkupsFiducialNode_15,6.032,-14.321,-8.436,0.000,0.000,0.000,1.000,1,1,0,8,R inferior LMS,vtkMRMLScalarVolumeNode1
vtkMRMLMarkupsFiducialNode_16,-5.966,-14.255,-8.436,0.000,0.000,0.000,1.000,1,1,0,9,L inferior LMS,vtkMRMLScalarVolumeNode1
vtkMRMLMarkupsFiducialNode_17,-0.085,-26.505,6.147,0.000,0.000,0.000,1.000,1,1,0,10,culmen,vtkMRMLScalarVolumeNode1
vtkMRMLMarkupsFiducialNode_18,0.000,-5.005,-6.566,0.000,0.000,0.000,1.000,1,1,0,11,intermammillary sulcus,vtkMRMLScalarVolumeNode1
vtkMRMLMarkupsFiducialNode_19,1.024,-5.255,-6.310,0.000,0.000,0.000,1.000,1,1,0,12,R MB,vtkMRMLScalarVolumeNode1
vtkMRMLMarkupsFiducialNode_20,-1.280,-5.255,-6.353,0.000,0.000,0.000,1.000,1,1,0,13,L MB,vtkMRMLScalarVolumeNode1
vtkMRMLMarkupsFiducialNode_21,-0.256,-15.255,2.478,0.000,0.000,0.000,1.000,1,1,0,14,pineal gland,vtkMRMLScalarVolumeNode1
vtkMRMLMarkupsFiducialNode_22,3.072,0.495,8.280,0.000,0.000,0.000,1.000,1,1,0,15,R LV at AC,vtkMRMLScalarVolumeNode1
vtkMRMLMarkupsFiducialNode_23,-3.413,0.495,8.195,0.000,0.000,0.000,1.000,1,1,0,16,L LV at AC,vtkMRMLScalarVolumeNode1
vtkMRMLMarkupsFiducialNode_24,6.486,-12.542,8.747,0.000,0.000,0.000,1.000,1,1,0,17,R LV at PC,vtkMRMLScalarVolumeNode1
vtkMRMLMarkupsFiducialNode_25,-6.741,-12.542,8.622,0.000,0.000,0.000,1.000,1,1,0,18,L LV at PC,vtkMRMLScalarVolumeNode1
vtkMRMLMarkupsFiducialNode_7,-0.029,10.995,6.474,0.000,0.000,0.000,1.000,1,1,0,19,genu of CC,vtkMRMLScalarVolumeNode1
vtkMRMLMarkupsFiducialNode_27,-0.197,-15.567,3.713,0.000,0.000,0.000,1.000,1,1,0,20,splenium of CC,vtkMRMLScalarVolumeNode1
vtkMRMLMarkupsFiducialNode_28,12.042,-2.557,-13.129,0.000,0.000,0.000,1.000,1,1,0,21,R AL temporal horn,vtkMRMLScalarVolumeNode1
vtkMRMLMarkupsFiducialNode_29,-12.458,-2.680,-13.947,0.000,0.000,0.000,1.000,1,1,0,22,L AL temporal horn,vtkMRMLScalarVolumeNode1
vtkMRMLMarkupsFiducialNode_30,7.253,-4.180,-9.467,0.000,0.000,0.000,1.000,1,1,0,23,R superior AM temporal horn,vtkMRMLScalarVolumeNode1
vtkMRMLMarkupsFiducialNode_31,-7.828,-4.239,-9.967,0.000,0.000,0.000,1.000,1,1,0,24,L superior AM temporal horn,vtkMRMLScalarVolumeNode1
vtkMRMLMarkupsFiducialNode_32,8.064,-3.003,-12.967,0.000,0.000,0.000,1.000,1,1,0,25,R inferior AM temporal horn,vtkMRMLScalarVolumeNode1
vtkMRMLMarkupsFiducialNode_33,-8.417,-3.121,-13.467,0.000,0.000,0.000,1.000,1,1,0,26,L inferior AM temporal horn,vtkMRMLScalarVolumeNode1
vtkMRMLMarkupsFiducialNode_34,6.997,-18.371,3.161,0.000,0.000,0.000,1.000,1,1,0,27,R indusium griseum origin,vtkMRMLScalarVolumeNode1
vtkMRMLMarkupsFiducialNode_35,-7.475,-18.542,2.911,0.000,0.000,0.000,1.000,1,1,0,28,L indusium griseum origin,vtkMRMLScalarVolumeNode1
vtkMRMLMarkupsFiducialNode_36,14.275,-22.526,0.862,0.000,0.000,0.000,1.000,1,1,0,29,R ventral occipital horn,vtkMRMLScalarVolumeNode1
vtkMRMLMarkupsFiducialNode_37,-14.420,-23.026,0.345,0.000,0.000,0.000,1.000,1,1,0,30,L ventral occipital horn,vtkMRMLScalarVolumeNode1
vtkMRMLMarkupsFiducialNode_38,4.522,12.995,3.673,0.000,0.000,0.000,1.000,1,1,0,31,R olfactory sulcal fundus,vtkMRMLScalarVolumeNode1
vtkMRMLMarkupsFiducialNode_39,-4.864,12.995,3.587,0.000,0.000,0.000,1.000,1,1,0,32,L olfactory sulcal fundus,vtkMRMLScalarVolumeNode1
```

**Comment by kaitj** [comment]
> Hey @kaitj, I think you've already found one but for prosperity there are a bunch here including a macaqueMNI one https://github.com/afids/afids-macaca/blob/master/data/PHASE1_output_afid/macaqueMNI_MEAN.fcsv

That is the one I've added now as our .fcsv test case.

> Here is a test case that was failing for me on the default but not with a human template selected. Couldn't attach a csv file so including inline. Also uploaded on Slack.

 Just tried to running this through the validator and it returned a "Valid file"
![Screenshot from 2020-05-29 13-32-17](https://user-images.githubusercontent.com/21226986/83288278-13b2f500-a1b1-11ea-900c-7bd57a85e9af.png)

I did note though that if a file was previously uploaded and a new file is uploaded without changing the option, the page may appear as though it refreshed as the message is still the same. There might be a couple of ways around this, we could change the message as a pop up (though this is probably disruptive) or we could incorporate a time stamp. Any thoughts @tkkuehn ?
